### PR TITLE
64 examples add graal import usage

### PR DESCRIPTION
**Pull request by Adrian-Bielefeldt**
Added two examples showing how to convert from Graal Model objects to VLog4J objects. See [issue 64](https://github.com/knowsys/vlog4j/issues/64) for more details.

### CODE REVIEW

File: vlog4j-examples/pom.xml [hunk 1]
```diff
@@ -49,7 +54,16 @@
 			<version>2.7.16</version>
 		</dependency>
 		
-		
+		<dependency>
```
Comment (irina-dragoste) [inline vlog4j-examples/pom.xml]: there is already a dependency to graal-api declared in vlog4j-graal. Therefore, it is not needed here.

File: vlog4j-examples/pom.xml [hunk 2]
```diff
@@ -49,7 +54,16 @@
...
-		
+		<dependency>
+			<groupId>fr.lirmm.graphik</groupId>
+			<artifactId>graal-api</artifactId>
+			<version>1.3.1</version>
+		</dependency>
+    	<dependency>
+      		<groupId>fr.lirmm.graphik</groupId>
+      		<artifactId>graal-io-dlgp</artifactId>
+      		<version>1.3.1</version>
```
Comment (irina-dragoste) [inline vlog4j-examples/pom.xml]: use ${graal.version} property from parent pom

File: vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
+package org.semanticweb.vlog4j.examples.graal;
+
```
Comment (irina-dragoste) [inline vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java]: This file does not contain the license header. The header is automatically generated by running a maven build.

File: vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+import org.semanticweb.vlog4j.graal.GraalConjunctiveQueryToRule;
+import org.semanticweb.vlog4j.graal.GraalToVLog4JModelConverter;
+
+import fr.lirmm.graphik.graal.api.core.Atom;
+import fr.lirmm.graphik.graal.api.core.ConjunctiveQuery;
+import fr.lirmm.graphik.graal.api.core.Rule;
+import fr.lirmm.graphik.graal.io.dlp.DlgpParser;
+
+/**
+ * This example shows how facts can be imported from files in the DLGP/DLP
```
Comment (irina-dragoste) [inline vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java]: Please add a link to http://graphik-team.github.io/graal/doc/dlgp when referencing DLGP format.

File: vlog4j-examples/pom.xml (hunk 1 above)
Comment (Adrian-Bielefeldt) [reply vlog4j-examples/pom.xml]: Superfluous dependency removed.

File: vlog4j-examples/pom.xml (hunk 2 above)
Comment (Adrian-Bielefeldt) [reply vlog4j-examples/pom.xml]: Moved version number to parent pom and rewrote examples and graal pom accordingly.

File: vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java (hunk 3 above)
Comment (Adrian-Bielefeldt) [reply vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java]: Added license header.

File: vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java (hunk 4 above)
Comment (Adrian-Bielefeldt) [reply vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java]: Added hyperlink to the documentation.

**Pull request merged**